apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/DeployNewReplicaPanel.tsx

```diff
@@ -128,6 +152,7 @@ const DeployNewReplicaPanel = ({
   const computeAddons =
     addons?.available_addons.find((addon) => addon.type === 'compute_instance')?.variants ?? []
   const selectedComputeMeta = computeAddons.find((addon) => addon.identifier === selectedCompute)
+  const estComputeMonthlyCost = Math.floor((selectedComputeMeta?.price ?? 0) * 730) // 730 hours in a month
```

Comment:
Fine for now, API should really expose that though - 730 may not necessarily be correct. Good enough for now though